### ISSUE TITLE
Turn Leaving Earth - Equipment into a game of hot or cold

### ISSUE THREAD

**Issue by hexagonrecursion** [colobot/colobot#1780]
### Describe the feature

The first mission of the game "Leaving Earth - Equipment" is too difficult and not in a fun way. Some players end up wandering around the map unable to find the backpack. Maybe we should turn it into a [game of hot or cold](https://en.wikipedia.org/wiki/Hunt_the_thimble#Playing_with_Hot_or_Cold) - periodically notify the player whether they are closer to the backpack than last time or further away

**Comment by ScienceDiscoverer** [comment]
Yes, I tried to beat the game after decades of not playing it and this mission still frustrated me greatly.
